src/js/components/List/List.js

```diff
@@ -88,9 +82,16 @@ const List = React.forwardRef(
         }
       >
         <StyledList
-          ref={ref}
+          ref={ref || listRef}
```

Comment:
We should use `useForwardedRef` instead.

Reply:
Thanks! I wasn't aware of this hook.